src/componentsTest/ForumTest.js

```diff
@@ -0,0 +1,261 @@
+        try {
+            const res = await callApiLoadComments();
+            const parsed = JSON.parse(JSON.stringify(res.express));
+            //setComments(parsed);
+
+
+        } catch (error) {
+            //console.error(error);
+        }
+    }
+
+    const callApiLoadComments = async () => {
+        const url = `${REACT_APP_API_ENDPOINT}/getForumCommentsByForumID`;
+        console.log(url);
+
+        const response = await fetch(url, {
+            method: "POST",
+            headers: {
+                "Content-Type": "application/json",
+            },
+            body: JSON.stringify({
+                forumID: forumID
+            })
+        });
+        const body = await response.json();
+        if (response.status !== 200) throw Error(body.message);
+        return body;
+    }
+
+    //API - Delete your own comments 
+    const handleApiDeleteComment = (commentID) => {
+        callApiDeleteComment(commentID)
+            .then(res => {
+                console.log("callApiDeleteComment returned: ", res)
+            })
+        setTimeout(() => {
+            window.location.reload();
+        }, 100);
+    }
+
+    const callApiDeleteComment = async (commentID) => {
+        const url = `${REACT_APP_API_ENDPOINT}/deleteForumComment`;
+        console.log(url);
+
+        const response = await fetch(url, {
+            method: "POST",
+            headers: {
+                "Content-Type": "application/json",
+            },
+            body: JSON.stringify({
+                commentID: commentID
+            })
+        });
+        const body = await response.json();
+        if (response.status !== 200) throw Error(body.message);
+        return body;
+    }
+
+    return (
+        <div id="body">
+
+            <Box sx={{ position: 'absolute', top: 110, left: '50%', transform: 'translate(-50%, -50%)' }}>
+                <Typography
+                    variant="h4"
+                    gutterBottom
+                    component="div">
+                    Forums
+                </Typography>
+            </Box>
+
+            <Box sx={{ position: 'absolute', top: 150, left: '50%', transform: 'translateX(-50%)' }}>
+                {forums.map((event) => (
+                    <Card style={{ width: '1000px', marginBottom: '20px' }} key={event.id}>
+                        <CardContent>
+                            {/*<Link to={`/forum/${event.id}`} target="_blank">*/}
+                            <Typography variant="h5" component="div">
+                                {event.forumTitle}<br />
+                            </Typography>
+                            {/*</Link>*/}
+                            <Typography sx={{ mb: 1.5 }} color="text.secondary">
+                                Posted on {new Date(event.dateTime).toLocaleDateString()}<br />
+                            </Typography>
+                            <Typography sx={{ mb: 1.5 }} color="text.secondary">
```

Comment:
remove the `color=text.secondary` styling from all components as it throws a warning during test runs